### DISCUSSION TITLE
My keyboard can't work at all!!!!!

### DISCUSSION THREAD

**Discussion by MoeNyanSakura** [PlayCover/PlayCover#1097] [Q&A]
M2 Macbook air Sonoma14.0(bata) Playcover(3.0,2.50)

The keyboard can't work in the app,it can't control the character to move in the game.Although I have do all the settings!

**Comment by MEMORIEmusic** [comment]
Same as #1048. Read [my comment](https://github.com/PlayCover/PlayCover/issues/1048#issuecomment-1691960965).

**Comment by daoba2733** [comment]
Downgrading OS
